dxfeed/wrappers/subscription.py

```diff
@@ -20,19 +20,21 @@ class Subscription(object):
     date_time: str or datetime.datetime
         If present timed subscription will be created (conflated stream). For sting date format is following:
         %Y-%m-%d %H:%M:%S.%f. If None - stream subscription will be created (non-conflated). Default - None.
+    exact_format: bool
```

Comment:
Seems that part of this documentation should be moved to the `__init__` method. See [here](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) for details.